src/components/DropDownMenu.svelte

```diff
@@ -28,26 +28,26 @@
   }: Props = $props();
 </script>
 
-<div class="column freq-select gap-f8 w-full max-w-[460px]">
```

Comment:
All of the style changes here are because I accidentally merged the wrong thing in a rebase.